TortolaniHW03.Rmd

```diff
@@ -0,0 +1,158 @@
+---
+title: "HW03"
+author: "Ariana F. Tortolani"
+date: "7/20/2020"
+output: github_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+## Description 
+
+For this assignment I will be using the COVID-19 data from the NYT's Github Repository to generate these figures.
+
+Below I am loading important libraries that will be useful in creating my figures.
+
+```{r load libraries, echo = TRUE, include=TRUE}
+library("ggplot2")
+library("magrittr")
+library("readr")
+library("dplyr")
+```
+
+## Import the Data
+
+I am using the us-states.cvs dataset from the NYT.  First I went to the [NYT Github Repository for COVID-19 Data](https://github.com/nytimes/covid-19-data).  I forked this repository and then downloaded the specific file I was interested in, *"us-states.csv"*.  After downloading this to my computer I then uploaded it back to my HW03 Github repository by clicking the "add file" button in github, then I pulled the change back down to RStudio to work with this file.
+
+QUESTION:  I was trying to find a way to add this document directly from one github repository to another (i.e. going directly from the NYT repo to my HW03 repo without having to download the file to my computer and reupload).  I was not able to find a good way to do this and was wondering if anyone else had?
+
+```{r data, echo=TRUE}
+all_states <- read.csv("us-states.csv", stringsAsFactors = FALSE)
+```
+
+##Graph 1
+
+Evaluating the prevalence of COVID-19 in Rhode Island.
+
+```{r}
+#filter the data for only Rhode Island
+all_states %>%
+  filter(state == "Rhode Island") -> RI_data
+  
+#re-format the date col
+RI_data$date <- as.Date(RI_data$date)
+
+#create the plot 
+RI_plot <- ggplot(RI_data) +
+  geom_col(aes(date, cases), fill = "dodgerblue3") +
+  geom_col(aes(date, deaths), fill = "firebrick3") +
+  
+#add axis labels / title
+  labs (title = "Total Number of COVID-19 Cases in Rhode Island",
+            x = "Date (year / month / day)", 
+            y = "Total Number of Cases / Deaths") +
+
+#make easier to read
+  theme(axis.text.x = element_text(angle=90, vjust = 0.65),
+        plot.title = element_text(size = 14, face = "bold", hjust = 0.5),
+        legend.position = c(1,1)) +
+  scale_x_date(date_breaks = "1 weeks") +
+  
+#create legend
+  annotate("text", as.Date("2020-03-16"), 13500, label = "Cases =") +
+  annotate("rect", xmin = as.Date("2020-03-25"), xmax = as.Date("2020-04-01"),
+                   ymin = 13250, ymax = 13750,
+                   fill = "dodgerblue3") +
+  annotate("text", as.Date("2020-03-16"), 11500, label = "Deaths =") +
+  annotate("rect", xmin = as.Date("2020-03-25"), xmax = as.Date("2020-04-01"),
+                   ymin = 11250, ymax = 11750,
+                   fill = "firebrick3")
+
```

Comment:
I forgot about using annotate() to change the text of the legends, I'll have to remember this next time! Wasted 30min trying to modify my legends without success

Reply:
took me a while to think of this as well!  the hardest part (after thinking of it) was honestly figuring out the placement with xmax / xmin / ymax / ymin, a lot of just trial and error there until it looked about right.